plugins/polio/js/src/domains/Campaigns/MainDialog/usePolioDialogTabs.tsx

```diff
@@ -20,6 +20,20 @@ import { SubActivitiesForm } from '../SubActivities/SubActivitiesForm';
 import { useIsPolioCampaign } from '../hooks/useIsPolioCampaignCheck';
 import { Tab } from './PolioDialogTabs';
 
+function determineSubActivityTabTooltip(
+    formik: FormikProps<CampaignFormValues>,
+    formatMessage: any,
+) {
```

Comment:
We prefer to use "fat arrow" functions: 

```suggestion
const  determineSubActivityTabTooltip = (
    formik: FormikProps<CampaignFormValues>,
    formatMessage: (message: IntlMessage) => string,
) :string => {
```

You could even make the function a hook to avoid having to pass `formatMessage`:

```typescript
const useSubActivityTabTooltip(
    formik: FormikProps<CampaignFormValues>,
) {
    if (
        formik.values.id &&
        formik.values.separate_scopes_per_round !==
            formik.initialValues.separate_scopes_per_round
    ) {
        return formatMessage(MESSAGES.subActivitiesLockedScopeChange);
    }
    return formatMessage(MESSAGES.subActivitiesUnlockConditions);
}
```

Reply:
and `const { formatMessage } = useSafeIntl()`